samples/tacos/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application
+        android:supportsRtl="true"
+        android:theme="@style/TacoTheme">
+        <activity
```

Comment:
nit: any reason not to use the configChanges param to take full control here?

Reply:
I briefly considered going that route but didn't explore it. I don't see any reason not to do that.

I updated to use `configChanges` and removed all uses of `rememberSaveable` and `Parcelable`: [06b1250](https://github.com/slackhq/circuit/pull/496/commits/06b1250f27c1e130a3063695e3ee2086a0bc8ed4)